### ISSUE TITLE
"use client" error from @emotion/react

### ISSUE THREAD

**Issue by kevinpastor** [coder-xiaotian/swc-useclient#1]
I've setup my project for @mui/material following the example but I'm still getting the same error as before. I'm using PNPM if this changes anything.

Here the relevant logs:

```txt
- error node_modules\.pnpm\@emotion+react@11.11.1_@types+react@18.2.14_react@18.2.0\node_modules\@emotion\react\dist\emotion-element-6bdfffb2.esm.js (14:41) @ React
- error createContext only works in Client Components. Add the "use client" directive at the top of the file to use it. Read more: https://nextjs.org/docs/messages/context-in-server-component
    at eval (webpack-internal:///(sc_server)/./node_modules/.pnpm/@emotion+react@11.11.1_@types+react@18.2.14_react@18.2.0/node_modules/@emotion/react/dist/emotion-element-6bdfffb2.esm.js:34:92)
    at (sc_server)/./node_modules/.pnpm/@emotion+react@11.11.1_@types+react@18.2.14_react@18.2.0/node_modules/@emotion/react/dist/emotion-element-6bdfffb2.esm.js (D:\Programming\lq\.next\server\app\page.js:11938:1)
    at __webpack_require__ (D:\Programming\lq\.next\server\webpack-runtime.js:33:42)
    at eval (webpack-internal:///(sc_server)/./node_modules/.pnpm/@emotion+styled@11.11.0_@emotion+react@11.11.1_@types+react@18.2.14_react@18.2.0/node_modules/@emotion/styled/base/dist/emotion-styled-base.esm.js:8:72)
    at (sc_server)/./node_modules/.pnpm/@emotion+styled@11.11.0_@emotion+react@11.11.1_@types+react@18.2.14_react@18.2.0/node_modules/@emotion/styled/base/dist/emotion-styled-base.esm.js (D:\Programming\lq\.next\server\app\page.js:11982:1)
    at __webpack_require__ (D:\Programming\lq\.next\server\webpack-runtime.js:33:42)
    at eval (webpack-internal:///(sc_server)/./node_modules/.pnpm/@emotion+styled@11.11.0_@emotion+react@11.11.1_@types+react@18.2.14_react@18.2.0/node_modules/@emotion/styled/dist/emotion-styled.esm.js:5:95)
    at (sc_server)/./node_modules/.pnpm/@emotion+styled@11.11.0_@emotion+react@11.11.1_@types+react@18.2.14_react@18.2.0/node_modules/@emotion/styled/dist/emotion-styled.esm.js (D:\Programming\lq\.next\server\app\page.js:11993:1)
    at __webpack_require__ (D:\Programming\lq\.next\server\webpack-runtime.js:33:42)
    at eval (webpack-internal:///(sc_server)/./node_modules/.pnpm/@mui+styled-engine@5.13.2_@emotion+react@11.11.1_@emotion+styled@11.11.0_react@18.2.0/node_modules/@mui/styled-engine/node/index.js:44:38) {
  name: 'TypeError',
  digest: undefined
}
null
```

Is there anything I'm missing out? I've tried adding `@emotion/react` to the `include` array of the configuration, without success.

**Comment by coder-xiaotian** [comment]
You are using Windows, did you try to include: ["@emotion\react"] rather than "@emotion/react"?

**Comment by coder-xiaotian** [comment]
I fixed this problem in 1.0.4， you can use `include: ["@emotion/react"]` directly.

**Comment by kevinpastor** [comment]
In the end, the update seems to have fixed my issue. I was indeed on Windows. My current setup looks a bit like this:

```js
{
    // ...
    experimental: {
        swcPlugins: [
            [
                "use-client",
                {
                    include: [
                        "@mui/material",
                        "@mui/icons-material"
                    ]
                }
            ]
        ]
    }
}
```
